scipy/sparse/_coo.py

```diff
@@ -99,23 +99,26 @@ def __init__(self, arg1, shape=None, dtype=None, copy=False):
 
     @property
     def row(self):
-        return self.indices[0]
+        if self.ndim > 1:
+            return self.indices[0]
+        return np.zeros(self.nnz, dtype=self.indices[0].dtype)
```

Comment:
I'd prefer:

```
return self.indices[-2] if self.ndim > 1 else np.zeros_like(self.col)
```

Reply:
Good...let's use index `-2` throughout and inline if/else in cases like this.